### ISSUE TITLE
unset all player 1,2,3,4 controller options at startup 

### ISSUE THREAD

**Issue by themailboxhere** [meleu/RetroPie-joystick-selection#21]
I have a system that had arcade sticks (geeekpie usb controller card) in which I also have 2 PS3 BT controllers as well as an 8bitdo N64 controller. sometimes I want to use the arcade sticks, sometimes I would rather use something else. your script/app is great for putting the option to change what controllers you use globally or per system up front so it is faster to manage but something I have noticed is that if you go and unset all the emulator specific player 1,2,3,4s, then set your global(all) playr controllers and proceed to play a game that retropie sets that specific emulators player 1,2,3,4 controller override  to whatever you used last... next time you boot up the pie, the specific emulator player controller config remembers the last controller you used as player 1,2,3,4... thus, changing the global(all) player controller config holds no value over the specified system. at this point you have to go into the specific system and "unset" players 1,2,3,4.... kind of a pain when you have multiple systems...

Is there any way way you could add an option to clear "unset" all emulator specific player 1,2,3,4 joystick choices on bootup? that would make the global (all) joystick config work so much smoother for people who have a multi-controller setup.

Obviously this should be an optional toggle switch of some sort as some people do not have a plethora of controllers they want to throw at the pie but it for sure would be greatly appreciated by many if this was an option.... Clearing the presets for 14 systems can be a pain every time you want to switch what controller you want to use...

Thank you for your time and experience in the project, you ROCK :)  please consider my option.

**Comment by meleu** [comment]
Not sure if I followed your scenario very well. But I think your request is what the title says. In other words:

add an option to unset everything at startup.

Am I right?

**Comment by themailboxhere** [comment]
actually, I figured out what to do. I reset all of the configs per system.
made a new folder called clean_configs and put all of the files in this
folder (in subfolders with the console names of course because all of the
files have the same name. Then I added a script to overwrite the config
files with the clean ones. so, all I have to do is set the global joystick
config to whatever I want to use and it works for every system.
